magento2-nginx/fpm-7.0/etc/nginx/sites-available/default-50_site_final_match.conf

```diff
@@ -0,0 +1,4 @@
+# Banned locations (only reached if the earlier PHP entry point regexes don't match)
+location ~* (\.php$|\.htaccess$|\.git) {
```

Comment:
`(/|^)([^\]*.php|\.(ht|git)[^\]*)(/|$)` is more secure (e.g. blocks /index.php/something and doesn't block /somethingabout.git/.